src/Modules/Footer/FooterComponent.js

```diff
@@ -4,37 +4,43 @@
 import React from 'react';
 import LogoComponent from '../General/LogoComponent';
 import { withRouter } from 'react-router';
-import { useHistory } from 'react-router-dom';
-const FooterComponent = (props) => {
-    let history = useHistory();
-    return (
-    	<div className="container pt-50">
-                <div className="row">
-                    <LogoComponent/>
-                    <div className="col-lg-6 col-xl-6">
-                        <div className="row">
-                            <div className="col-md-6">
-                                <span className="list-title">FIND RESOURCES</span>
-                                <ul>
-                                    <li><a onClick={()=>history.push('/freshtrak-about')}>About FreshTrak</a></li>
-                                </ul>
-                            </div>
-                            <div className="col-md-6">
-                                <span className="list-title">FOR FOODBANKS</span>
-                                <ul>
-                                    <li><a onClick={()=>history.push('freshtrak-working')}>Working with FreshTrak</a></li>
-                                </ul>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <div className="row pt-2 pb-3">
-                    <div className="col-md-12">
-                        <p className="copy-right">© 2020 FreshTrak</p>
-                    </div>
-                </div>
-            </div>
-    )
-};
+import { Link } from 'react-router-dom';
+import { RENDER_URL } from '../../Utils/Urls'
+const FooterComponent = () => (
+  <div className="container pt-50">
+    <div className="row">
+      <LogoComponent/>
+      <div className="col-lg-6 col-xl-6">
+        <div className="row">
+          <div className="col-md-6">
+            <span className="list-title">FIND RESOURCES</span>
+            <ul>
+              <li>
+                <Link to={RENDER_URL.FRESHTRAK_ABOUT}>
+                  About FreshTrak
+                </Link>
```

Comment:
Use the provide `react-router` `Link` component instead of pushing to history. It is more accessible and cleaner.